package.json

```diff
@@ -84,6 +87,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-intl": "6.5.5",
+    "rollup": "4.8.0",
```

Comment:
Why do we need this?

Reply:
the install was complaining that it was missing, 
![Screenshot 2023-12-13 at 10 47 36](https://github.com/SonarSource/echoes-react/assets/97296331/3b4f3891-46bf-4c47-8ec7-627382ddfe4a)

but perhaps it's not needed (confusing why it would complain though) as it mentions if a plugin is written as rollup compatible, it doesn't need rollup. I guess if the license header is working correctly we don't need it 👍🏻 

https://vitejs.dev/guide/api-plugin#rollup-plugin-compatibility

Reply:
I would say it's worth keeping it since we have some stuff depending on it, instead of having the warning and always coming to questioning ourselves why we see it...